src/main/java/ro/developmentfactory/thedrones/controller/DroneControllerPosition.java

```diff
@@ -0,0 +1,47 @@
+package ro.developmentfactory.thedrones.controller;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import ro.developmentfactory.thedrones.controller.dto.DroneStatusResponse;
+import ro.developmentfactory.thedrones.service.DronePositionService;
+
+@RestController
+@RequestMapping("/drones/status")
+public class DroneControllerPosition {
+
+    private final DronePositionService dronePositionService;
+    private static final Logger log = LoggerFactory.getLogger(DroneControllerPosition.class);
+
+    public DroneControllerPosition(DronePositionService dronePositionService) {
+        this.dronePositionService = dronePositionService;
+    }
+
+    @PostMapping("/turnLeft")
+    public ResponseEntity<DroneStatusResponse> turnLeft(@RequestBody DroneStatusResponse droneStatusResponse) {
```

Comment:
don't use a **response** `DroneStatusResponse` when you should expect a **request**! in fact, you don't need request here - anyway you are only using its id and you can get it from the `@PathVariable` if you follow `POST /drones/{id}/turn/L` + `POST /drones/{id}/turn/R`. this applies to all methods in this controller